kount/client.py

```diff
@@ -79,7 +79,12 @@ def process(self, params):
                 logger.debug(error)
                 raise ValueError(error)
         else:
-            logger.debug("process json: %s", req_json)
+            roundtrip = request.elapsed.total_seconds()
+            merc = headers_api.get("X-Kount-Merc-Id", None)
```

Comment:
MERC is OK to be retrieved from parameters also; and they both can't be null/empty, so remove the `None` if there's such method in python